catroid/src/main/java/org/catrobat/catroid/utils/Utils.java

```diff
@@ -431,18 +397,15 @@ public static Pixmap getPixmapFromFile(File imageFile) {
 		return pixmap;
 	}
 
-	public static String getUniqueProjectName() {
-		String projectName = "project_" + System.currentTimeMillis();
-		while (XstreamSerializer.getInstance().projectExists(projectName)) {
-			projectName = "project_" + System.currentTimeMillis();
-		}
-		return projectName;
-	}
-
 	public static boolean isDefaultProject(Project projectToCheck, Context context) {
 		try {
-			Project defaultProject = DefaultProjectHandler
-					.createAndSaveDefaultProject(getUniqueProjectName(), context);
+			String uniqueProjectName = "project_" + System.currentTimeMillis();
+
+			while (XstreamSerializer.getInstance().projectExists(uniqueProjectName)) {
+				uniqueProjectName = "project_" + System.currentTimeMillis();
+			}
+
+			Project defaultProject = DefaultProjectHandler.createAndSaveDefaultProject(uniqueProjectName, context);
```

Comment:
Ah, you inlined the `getUniqueProjectName` method so developer won't use it (not directly nor indirectly) :+1: